i18n/i18n/main.go

```diff
@@ -117,7 +117,7 @@ func main() {
 	sort.Slice(keys, func(i, j int) bool {
 		return txt.NaturalLess(keys[i], keys[j], true)
 	})
-	out, err := os.Create(outPath)
+	out, err := os.OpenFile(outPath, os.O_RDWR|os.O_CREATE|os.O_TRUNC, 0600)
```

Comment:
Again, this is for a very specific purpose -- in this case, for a file to be included in a distribution for an application. The standard settings of 0666 were fine, although reducing them to 0644 would be acceptable, too. 0600 is not.